### PR TITLE
Added QA environment info to README #893

### LINKED ISSUE #893: Update readme with QA env info

### PR DESCRIPTION
**Pull request by talentedunicorn**
### What github issue is this PR for, if any?
Resolves #893 

### What changed, and why?
Added QA env information to README

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### How is this tested? (please write tests!) 💖💪
N/A

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,14 @@ Deploy to Staging is manual. Training of new users is done in staging.
 
 https://casa-r4g-staging.herokuapp.com/
 
+### QA environment
+
+QA is done on the staging servers
+
+- https://casa-r4g-staging.herokuapp.com/
```
Comment (jmkoni) [inline README.md:189]: That is our staging server. QA is actually https://casa-qa.herokuapp.com/

**Review by jmkoni — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (talentedunicorn) [reply README.md:189]: @jmkoni thanks i'll update that. Do the same credentials apply to QA server?

Comment (jmkoni) [reply README.md]: Yes

Comment (talentedunicorn) [reply README.md]: 🙃 this should be good to go then

**Review by compwron — APPROVED** [review]
:)

**Comment by compwron** [comment]
You are invited to the CASA asynchronous retrospective-and-futurespective! Please take a look at https://bit.ly/casaretro (no login required) and add at least one note! Even if it's just "I was here and you are cool" or "omg y'all why do you keep messaging me" we want to hear it. <3
Next week Wednesday 18 November @ 6pm Pacific time we will Zoom about the board and talk about themes and takeaways. (Zoom link in our calendar at http://bit.ly/casacal or slack #casa channel https://rubyforgood.herokuapp.com/ )
Please ESPECIALLY add lots of notes if you won't be able to make it on Wednesday to talk!
* Note: the board will be monitored for spam; message me or any team lead if you see a problem
https://bit.ly/casaretro

**Pull request merged**